templates/cli/lib/commands/init.js.twig

```diff
@@ -0,0 +1,261 @@
+const fs = require("fs");
+const path = require("path");
+const childProcess = require('child_process');
+const { Command } = require("commander");
+const inquirer = require("inquirer");
+const { projectsCreate } = require("./projects");
+const { storageCreateBucket } = require("./storage");
+const { messagingCreateTopic } = require("./messaging");
+const { functionsCreate } = require("./functions");
+const { databasesCreateCollection } = require("./databases");
+const ID = require("../id");
+const { localConfig, globalConfig } = require("../config");
+const {
+    questionsCreateFunction,
+    questionsCreateBucket,
+    questionsCreateMessagingTopic,
+    questionsCreateCollection,
+    questionsInitProject
+} = require("../questions");
+const { success, log, error, actionRunner, commandDescriptions } = require("../parser");
+const { accountGet } = require("./account");
+const { loginCommand } = require("./generic");
+
+const initProject = async () => {
+    let response = {};
+
+    try {
+        if (globalConfig.getEndpoint() === '' || globalConfig.getCookie() === '') {
+            throw '';
+        }
+        await accountGet({
+            parseOutput: false
+        });
+    } catch (e) {
+        log('You must login first')
+        await loginCommand()
+    }
+
+    const answers = await inquirer.prompt(questionsInitProject)
+    if (!answers.project || !answers.organization) {
+        process.exit(1)
+    }
```

Comment:
Is this needed? Can we instead mark question as required? Please also do global search for process.exit and see if it can be fixed in other places too

Reply:
It's needed as the user can decide not to override the project, though the checking can be changed